mqtt/mqtt-broker/src/session.rs

```diff
@@ -96,6 +112,7 @@ impl ConnectedSession {
         self.state.publish_to(publication)
     }
 
+    #[allow(dead_code)]
     pub fn subscribe(
```

Comment:
Did we forget to remove this? Seem like `subscribe` should be updated to have the `subscribe_to` logic and one of these functions should be removed?

Reply:
Yes, should remove `subscribe`. Thanks!